openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedListV1.component.tsx

```diff
@@ -32,7 +31,7 @@ interface ActivityFeedListV1Props {
   activeFeedId?: string;
   hidePopover: boolean;
   isForFeedTab?: boolean;
-  emptyPlaceholderText: string;
+  emptyPlaceholderText: string | JSX.Element;
```

Comment:
Use `ReactNode` as it can handle these types `ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;`

take care of the same in other places as well.
```suggestion
  emptyPlaceholderText: ReactNode;
```